develop/application-devs/interact/endpoints.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Node Endpoints
+description: List of node endpoints and examples of use.  These endpoints can be used for development, RPC, or other purposes in which network access is required.
+---
+
+Ideally, one may run their own node when interacting with the
+[Polkadot network](https://polkadot.network/) via [Polkadot-JS Apps](https://polkadot.js.org/apps/)
+or other UIs and programmatic methods. Another option would be to connect to one of the several
+public endpoints provided by infrastructure and API service providers. For development convenience,
+[Parity Tech](https://www.parity.io/) maintains archive nodes for Polkadot, Kusama, and their test
+networks with public endpoints. These endpoints can be used with
+[Polkadot-JS API](https://polkadot.js.org/docs/api) to interact with their respective chains. The
+tables below list these endpoints.
+
+### Network Endpoints
+
+Endpoints for all production and test networks are listed on the
+[Polkadot-JS UI](https://polkadot.js.org/apps/#/accounts) which are accessed from
+[here](https://github.com/polkadot-js/apps/tree/master/packages/apps-config/src/endpoints).
+Endpoints for Polkadot Relay Chain and Kusama Relay Chain, parachains, and Paseo test network are
+maintained by the community. System Chains as well as Westend and Rococo test network endpoints
+maintained by Parity Technologies are listed below (with the exception of [Paseo](https://github.com/paseo-network), which is maintained by the wider community):
+
+
+=== "Polkadot"
+    | Network      | WSS Endpoint                               |
+    | ------------ | ------------------------------------------ |
+    | Asset Hub    | wss://polkadot-asset-hub-rpc.polkadot.io   |
```

Comment:
```suggestion
    | Asset Hub    | [wss://polkadot-asset-hub-rpc.polkadot.io](wss://polkadot-asset-hub-rpc.polkadot.io){target=\_blank}   |
```

I'd add this reference so users can easily copy and paste the endpoint. Not sure if this is the best way to handle though